extensions/sql-migration/src/constants/strings.ts

```diff
@@ -770,7 +774,7 @@ export function SERVICE_ERROR_NOT_READY(serviceName: string, error: string): str
 		error);
 }
 export function SERVICE_READY(serviceName: string, host: string): string {
-	return localize('sql.migration.service.ready', "Azure Database Migration Service '{0}' is connected to self-hosted integration runtime running on the node - {1}", serviceName, host);
+	return localize('sql.migration.service.ready', "Azure Database Migration Service '{0}' is connected to self-hosted integration runtime running on node(s) - {1}\n\nFor improved performance and high Availability, you can register additional nodes. See below for registration instructions.", serviceName, host);
 }
```

Comment:
lower case?

Reply:
DMS is the official product name, will leave it capitalized

Reply:
ah, I'm assuming you meant "Availability" - will fix that, good catch